app/command/put_content_with_links.rb

```diff
@@ -1,5 +1,11 @@
 class Command::PutContentWithLinks < Command::BaseCommand
   def call
+    if content_item[:content_id]
+      create_or_update_live_content_item!
+      create_or_update_draft_content_item!
+      create_or_update_links!
```

Comment:
Does this sequential set of calls maintain the single transaction model we need?